Project.toml

```diff
@@ -4,10 +4,12 @@ authors = ["Simon Kornblith <simon@simonster.com>"]
 version = "1.2.2"
 
 [compat]
+Aqua = "0.5.5"
```

Comment:
This package is so incredibly small and simple that I'd prefer we don't add extra dependencies even if they're only for testing. It seems vanishingly unlikely that this package would encounter any of the issues that Aqua is able to identify.